### PR TITLE
Fingerbank Upstream DB image builder

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Build a docker image that contains the Fingerbank upstream DB so that it can be mounted in k8s deployments
This only creates the image, we'll need to setup a schedule that publishes it on a regular basis to keep it up to date

# Impacts
Gitlab builds

# Delete branch after merge
YES

### CODE REVIEW

File: containers/fingerbank-db/Dockerfile
Comment (nqb) [inline containers/fingerbank-db/Dockerfile]: I suggest to add a retry option like we have during build of Fingerbank perl client:

https://github.com/fingerbank/perl-client/blob/e1ae4b2d6aacf8c90c18b1f3e82ec8dfc449cfe6/Makefile#L135-L137

**Review by nqb — CHANGES_REQUESTED** [review]

**Comment by nqb** [comment]
Two remarks:
- regarding schedule to build this image, at which frequency should we schedule it ?

I'm asking because if it's a different schedule than other images, it could be interesting to create a dedicated job for that image in place of reusing a job shared by several images (haproxy-portal at the moment)

- Before merging, I think we should adjust build of https://github.com/fingerbank/perl-client/ to remove DB from packaging.

**Comment by julsemaan** [comment]
> * regarding schedule to build this image, at which frequency should we schedule it ?
> 
> I'm asking because if it's a different schedule than other images, it could be interesting to create a dedicated job for that image in place of reusing a job shared by several images (haproxy-portal at the moment)

The DB is updated on a daily basis so 24h would make sense

> * Before merging, I think we should adjust build of https://github.com/fingerbank/perl-client/ to remove DB from packaging.

The intent (atm) is to have this be used only in k8s environment and leave the classic PF untouched where it gets a DB at install and updates it via pfcron. That is going to stay true for 12.0 until the Fingerbank integration is reworked to support contenarization. This just paves the way for that but doesn't actually do anything.

**Comment by julsemaan** [comment]
I added the retries and launched a pipeline to test it out: https://gitlab.com/inverse-inc/packetfence/-/pipelines/555111306

I think that if it passes we're good to merge unless @nqb has other adjustments he can suggest

**Review by nqb — APPROVED** [review]

**Comment by nqb** [comment]
I adjusted weekly schedule on `devel` to rebuild all container images each day.

**Pull request merged**
